src/main/java/uk/gov/hmcts/reform/slc/config/FtpConfiguration.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.slc.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.hmcts.reform.slc.model.FtpConfigProperties;
+
+@Configuration
+public class FtpConfiguration {
+
+    @Value("${ftp.hostname}")
+    private String hostname;
+
+    @Value("${ftp.port}")
+    private int port;
+
+    @Value("${ftp.fingerprint}")
+    private String fingerprint;
+
+    @Value("${ftp.user}")
+    private String username;
+
+    @Value("${ftp.keys.public}")
+    private String publicKey;
+
+    @Value("${ftp.keys.private}")
+    private String privateKey;
+
+    @Value("${ftp.target-folder}")
+    private String targetFolder;
+
+    @Value("${ftp.reports-folder}")
```

Comment:
we can define prefix in configuration file when there is a bunch of vals in use. and since it's component related config - makes sense to simplify the read

Reply:
good point